### PR TITLE
Azure Event Hubs Source Beta

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

This pull request adds new Azure Event Hub source (beta). This will eventually replace C2C version.

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205849078823071

### CODE REVIEW

File: docs/send-data/hosted-collectors/microsoft-source/ms-azure-event-hubs-source.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+---
+id: ms-azure-event-hubs-source
+title: Azure Event Hubs for Logs - Beta
+description: The Azure Event Hubs Source provides a secure endpoint to receive data from Azure Event Hubs.
+---
+
```
Comment (ajaiswals) [inline docs/send-data/hosted-collectors/microsoft-source/ms-azure-event-hubs-source.md:6]: Can we add the Beta tag, @kimsauce 

<p><a href="/docs/beta"><span className="beta">Beta</span></a></p>

Comment (kimsauce) [reply docs/send-data/hosted-collectors/microsoft-source/ms-azure-event-hubs-source.md:6]: @ajaiswals thank you so much - great catch!

File: sidebars.ts [hunk 2]
```diff
@@ -2620,7 +2620,7 @@ integrations: [
             'cse/schema/parsing-patterns',
             'cse/schema/parser-troubleshooting-tips',
             'cse/schema/username-and-hostname-normalization',
-          
+
```
Comment (kimsauce) [inline sidebars.ts:2623]:
```suggestion
          
```

File: sidebars.ts [hunk 3]
```diff
@@ -2828,4 +2828,4 @@ integrations: [
       ],
     },
   ],
-}
\ No newline at end of file
+}
```
Comment (kimsauce) [inline sidebars.ts:2831]:
```suggestion
}

```

File: sidebars.ts [hunk 4]
```diff
@@ -2828,4 +2828,5 @@ integrations: [
...
+
```
Comment (kimsauce) [inline sidebars.ts:2832]:
```suggestion
```

**Review by jpipkin1 — CHANGES_REQUESTED** [review]
@kimsauce, the content of the new "Azure Event Hubs for Logs - Beta" article looks like it's copied directly from the "[Azure Event Hubs Source](https://help.sumologic.com/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/azure-event-hubs-source)" article. Can you add any language to the beta article that differentiates how it differs from "Azure Event Hubs Source"?

**Comment by ajaiswals** [comment]
@kimsauce @jpipkin1 

We can add a statement : 

This source over comes the documented challenge of Azure C2C source scalability.

"Collecting data from Azure Event Hubs using this Cloud-to-Cloud collection source has a supported throughput limit of 1MB/S (86GB/day) for a named Event Hub egress rate"


Reference of existing source : https://help.sumologic.com/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/azure-event-hubs-source/

**Comment by ajaiswals** [comment]
@kimsauce @jpipkin1  - I think we should go ahead and publish the doc without the statement on the change. 

I am working with engineering to get some comparison number with respect to scale.  I will make those edits and submit a PR.

Can we merge the PR so we can add the Doc link to the Beta Announcement Page.

**Review by jpipkin1 — APPROVED** [review]
That sounds good, @ajaiswals. 

@kimsauce, I approved the PR.

**Comment by jpipkin1** [comment]
@kimsauce, I fixed a broken link that was causing the deploy test to fail with commit "[Fix broken link in FedRAMP article](https://github.com/SumoLogic/sumologic-documentation/pull/3279/commits/40c96c27ed0a7b1c5657149a88c5c15c1095741c)".

**Comment by kimsauce** [comment]
> @kimsauce, I fixed a broken link that was causing the deploy test to fail with commit "[Fix broken link in FedRAMP article](https://github.com/SumoLogic/sumologic-documentation/pull/3279/commits/40c96c27ed0a7b1c5657149a88c5c15c1095741c)".

You beat me to it, thank you!!

**Pull request merged**